examples/gno.land/p/demo/boards/misc.gno

```diff
@@ -0,0 +1,40 @@
+package boards
+
+import (
+	"strconv"
+	"strings"
+)
+
+func padLeft(str string, length int) string {
```

Comment:
this is not used anywhere.

Reply:
Just copied this one as well. Seems like it wasn't used in the original implementation.  

Removed!